src/models/participantEndpoint/facade.js

```diff
@@ -125,7 +127,20 @@ exports.sendErrorToParticipant = async (participantName, endpointType, errorInfo
     }
 
     Logger.debug(`participant endpoint url: ${requesterErrorEndpoint} for endpoint type ${endpointType}`)
-    await Util.Request.sendRequest(requesterErrorEndpoint, clonedHeaders, clonedHeaders[Enums.Http.Headers.FSPIOP.SOURCE], clonedHeaders[Enums.Http.Headers.FSPIOP.DESTINATION], Enums.Http.RestMethods.PUT, errorInformation, span)
+    let jwsSigner
+    if (Config.JWS_SIGN && clonedHeaders[Enums.Http.Headers.FSPIOP.SOURCE] === Config.FSPIOP_SOURCE_TO_SIGN) {
+      // We need below 2 headers for JWS
+      clonedHeaders[Enums.Http.Headers.FSPIOP.HTTP_METHOD] = clonedHeaders[Enums.Http.Headers.FSPIOP.HTTP_METHOD] || Enums.Http.RestMethods.PUT
```

Comment:
I would make an if here, even if it takes more lines, as it would make the code a bit more readable. Because when I first parsed it, I had to focus to realize that the two values are the same.
I don't insist about it though.